client/landing/components/Footer.tsx

```diff
@@ -81,9 +81,9 @@ const Footer = () => {
         </div>
         <div className="flex flex-col items-center justify-center basis-1/3">
           {/* <a href="https://dashboard.kzilla.xyz" target="_blank"> */}
```

Comment:
oh this can be uncommented with the link pointing to https://dash.kzilla.xyz and remove the `target` attribute